src/components/themed/WalletProgressIcon.js

```diff
@@ -36,6 +36,11 @@ export class WalletProgressIconComponent extends React.PureComponent<Props, Stat
         this.setState({ isDone: true })
       }, 5000)
     }
+    if (this.props.progress === 0) {
+      setTimeout(() => {
+        this.setState({ isDone: false })
+      }, 5000)
```

Comment:
Five seconds seems like overkill. Please set to 500 ms here as well as on line 37